### ISSUE TITLE
0.7.8 fails to install via pip - missing requirements.txt

### ISSUE THREAD

**Issue by rsmichaeldunn** [ringcentral/ringcentral-python#15]
```
$ pip install --no-cache-dir ringcentral==0.7.8
Collecting ringcentral==0.7.8
  Downloading ringcentral-0.7.8.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/0j/s3120gnd50zd0nqy1bknhclw0000gp/T/pip-build-YpAq3C/ringcentral/setup.py", line 15, in <module>
        install_requires=[i.strip() for i in open('requirements.txt').readlines()],
    IOError: [Errno 2] No such file or directory: 'requirements.txt'
```

versus 0.7.7:
```
$ pip install --no-cache-dir ringcentral==0.7.7
Collecting ringcentral==0.7.7
  Downloading ringcentral-0.7.7.tar.gz
Requirement already satisfied: observable>=0.3.1 in ./Library/Python/2.7/lib/python/site-packages (from ringcentral==0.7.7)
Requirement already satisfied: pubnub==4.* in ./Library/Python/2.7/lib/python/site-packages (from ringcentral==0.7.7)
Requirement already satisfied: pycryptodome>=3.4.4 in ./Library/Python/2.7/lib/python/site-packages (from ringcentral==0.7.7)
Requirement already satisfied: requests>=2.7.0 in /Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages (from ringcentral==0.7.7)
Requirement already satisfied: pycryptodomex>=3.3 in ./Library/Python/2.7/lib/python/site-packages (from pubnub==4.*->ringcentral==0.7.7)
Requirement already satisfied: six>=1.10 in ./Library/Python/2.7/lib/python/site-packages (from pubnub==4.*->ringcentral==0.7.7)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages (from requests>=2.7.0->ringcentral==0.7.7)
Requirement already satisfied: certifi>=2017.4.17 in /Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages (from requests>=2.7.0->ringcentral==0.7.7)
Requirement already satisfied: urllib3<1.23,>=1.21.1 in ./Library/Python/2.7/lib/python/site-packages (from requests>=2.7.0->ringcentral==0.7.7)
Requirement already satisfied: idna<2.7,>=2.5 in ./Library/Python/2.7/lib/python/site-packages (from requests>=2.7.0->ringcentral==0.7.7)
Installing collected packages: ringcentral
  Running setup.py install for ringcentral ... done
Successfully installed ringcentral-0.7.7
```

Happens on a MacBook Pro on High Sierra, and on an AmazonLinux OS.

**Comment by kirill-konshin** [comment]
I will look into it right now

**Comment by rsmichaeldunn** [comment]
Thanks.  I didn't see anything in the 0.7.8 commit that seemed like it would cause it, so it really stumped me.  I pinned my requirement to 0.7.7 for now, but wanted to report what I was seeing.  Please let me know if I can/need to provide any additional info.

**Comment by kirill-konshin** [comment]
Yes, I have no explanation either, there's nothing new. I suspect that some upgrade of `setuptools` and `pypi` caused that.

Fixed in `0.7.9`. Pls verify.

**Comment by rsmichaeldunn** [comment]
Confirmed.  Thanks!
